### DISCUSSION TITLE
28C256 pin compatible with ROM card on RC20214

### DISCUSSION THREAD

**Discussion by aylen384** [EtchedPixels/FUZIX#1071] [Q&A]
https://github.com/EtchedPixels/FUZIX/blob/master/Kernel/platform/platform-rc2014-tiny/README.md recommends using 28C256, but is that one actually PIN compatible to the old EPROMS?

**Comment by EtchedPixels** [comment]
The only wire differences are that the 27Cxxx has pin 1 as Vpp and pin 27 as A14, whilst the 28C256 has pin 1 as A14 and pin 27 is /WE so needs to be tied high. On the bigger 27Cxxx parts pin 1 becomes A15.

Jumper 'A14' (as the PCB sees it) to Vcc on the connector block (P3/P4/P5) so /WE is tied high. Leave the usual 16K jumper off JP5 entirely so that A15 does the chip select.  Instead connect pin 2 of JP5 to pin 6 of P3 (so A14 on the bus goes to pin 1 which is A14 on the 28C256)

I think that's right, but mine is currently in a box as I'm redecorating (and a bit more) my office.
